sql/share/errmsg-utf8.txt

```diff
@@ -7777,7 +7777,26 @@ ER_XA_REPLICATION_FILTERS
 ER_CANT_OPEN_ERROR_LOG
   eng "Could not open file '%s' for error logging%s%s"
 
-<<<<<<< HEAD
+ER_GROUPING_ON_TIMESTAMP_IN_DST
+  eng "Grouping on temporal is non-deterministic for timezones having DST. Please consider switching to UTC for this query."
+
+ER_CANT_START_SERVER_NAMED_PIPE
+  eng "Can't start server : Named Pipe \"%s\" already in use."
+
+#
+#  End of 5.7 error messages.
+#
+
+#
+# Start of Percona Server 5.7 error messages
+#
+
+start-error-number 11000
```

Comment:
Do you think we should try to synchronize this with 8.0 Percona-specific error code numbers as well?
@georgelorch do you have any experience with this?

Reply:
I believe they are/should already be synced.  MyRocks specific code start at 12000 for both 5.7 and 8.0

Reply:
Errors in 8.0 are much more complicated:
```
# 5.6 error messages
start-error-number 1000

# Error numbers 2000 to 2999 are reserved for the client libraries
reserved-error-section 2000 2999

# 5.7 error messages
start-error-number 3000

# 8.0 error messages
start-error-number 3500

# Error numbers 5000 to 5999 are reserved for X Plugin. 
reserved-error-section 5000 5999

# Start of 8.0 error messages intended to be logged to the server error log.
start-error-number 10000

# Start of Percona Server 8.0 server error log messages
start-error-number 48000

# Start of Percona Server 8.0 error messages to be sent to client
start-error-number 49000

# Start of MyRocks specific messages to be sent to client
start-error-number 50000
```

Reply:
Errors in 5.7 are simple:
```
# 5.6 error messages
start-error-number 1000

# 5.7 error messages
start-error-number 3000

# Start of Percona Server 5.7 error messages
start-error-number 11000

# Start of MyRocks specific messages
start-error-number 12000
```